templates/plugin/cpu.conf.erb

```diff
@@ -6,5 +6,9 @@
 <% if scope.lookupvar('collectd::collectd_version_real') and scope.function_versioncmp([scope.lookupvar('collectd::collectd_version_real'), '5.6']) >= 0 -%>
   ReportNumCpu <%= @reportnumcpu %>
 <% end -%>
+<% if scope.lookupvar('collectd::collectd_version_real') and scope.function_versioncmp([scope.lookupvar('collectd::collectd_version_real'), '5.8']) >= 0 -%>
+    ReportGuestState <%= @reportgueststate %>
+    SubtractGuestState <%= @subtractgueststate %>
+<% end -%>
```

Comment:
are you interested in converting this erb template into epp? This is not required, but we would be very happy about it :)

Reply:
It's OK, the file small enought, I done it :)